pkg/services/store/object/grn.go

```diff
@@ -0,0 +1,14 @@
+package object
+
+func (x *GRN) Equals(b *GRN) bool {
```

Comment:
@toddtreece -- any suggestions for something better here?

Reply:
i did some testing, and it looks like this could be:

```go
return b != nil && *x == *b

Reply:
hymm -- I get the error:
```
invalid operation: cannot compare *x == *b (struct containing impl.MessageState cannot be compared)compiler[UndefinedOp](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#UndefinedOp)
```

Reply:
ah ok, sorry about that. i was testing in the golang playground without the protobuf related fields in the struct. what you have makes sense to me 👍

Reply:
This is why I originally avoided using the protobuf structs for the identifier -- having a simple object is so much easier to work with.  That said, i think the value of explicitly defined fields is worth the pain